cpp/ps0004/problem solve 4.cpp

```diff
@@ -2,22 +2,27 @@
 using namespace std;
 int main()
 {
+    // prompting user for number 1.5/1.5
     float num;
     cout << "Enter a number from 32 to 126:\n";
     cin >> num;
-    if ((num >= 32) && (num <=126)) {
-    char asciiChar = (char) num;
-    cout << "The character for " << num << " is: " << asciiChar ;
+
+    // checking invalidaty of user's input 4/4
+    // print the character 2/2
+    if (num!=(int)num) {
+        cout << "Invalid a floating point number\n";
     }
-    else if (num!=(int)num) {
-    cout << "Invalid a floating point number\n";
+    else if ((num >= 32) && (num <=126)) {
+        char asciiChar = (char) num;
+        cout << "The character for " << num << " is: " << asciiChar ;
     }
     else if (num==0) {
-    cout << "Invalid character/s or zero value\n";
+        cout << "Invalid character/s or zero value\n";
     }
     else {
-    cout << "Invalid number is out of the range\n";
+        cout << "Invalid number is out of the range\n";
     }
     
     return 0;
-}
\ No newline at end of file
+}
+// great work 😎 8.5/10 ⭐
```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors, bugs or warnings while compilation and execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case

you can check the [problem solution](https://onlinegdb.com/9x6wLTlkK) for more